frontend/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "frontend",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "lint:ci": "eslint \"{src,apps,libs,test}/**/*.ts\"",
```

Comment:
Similar commands here. The backend uses `lint:ci` since `lint` had a auto --fix option.